documentation/documentation.html

```diff
@@ -177,5 +188,22 @@
           obj.contentWindow.document.documentElement.scrollHeight + "px";
       }
     </script>
+    <script>
+      document.querySelectorAll("#toggleBtn").forEach((item) =>
+        item.addEventListener("click", () => {
+          let navMenu = document.querySelector(".navbar-items");
+          console.log(navMenu);
+          navMenu.classList.toggle("navbar-list-res");
+        })
+      );
+      document.querySelectorAll(".nav-menu-mobile").forEach((item) => {
+        item.addEventListener("click", () => {
+          console.log("I am cliked", item);
+          let navMenu = document.querySelector(".navbar-items");
```

Comment:
If possible please remove console logs. You can add them again while adding improvements

Reply:
done